tests/conftest.py

```diff
@@ -5,6 +5,7 @@
 import pytest
 
 from hivemind.utils import get_logger
+from hivemind.utils.mpfuture import MPFuture
```

Comment:
Import from utils as well?

Reply:
In general, I'd argue for avoiding importing from `hivemind.utils` directly to avoid having one module with a lot of mixed-purpose stuff and the corresponding long import line.

To follow this, I've changed `from hivemind.utils import get_logger` above to `from hivemind.utils.logging import get_logger`.